lib/service/service.go

```diff
@@ -2587,12 +2590,13 @@ func (process *TeleportProcess) initUploaderService() error {
 	// safely return.
 	if conn == nil {
 		for _, localService := range types.LocalServiceMappings() {
-			if localService != types.RoleAuth && process.instanceRoleExpected(localService) {
+			if localService != types.RoleAuth && localService != types.RoleMDM && process.instanceRoleExpected(localService) {
```

Comment:
nit: Can we update the comment 

> // The only circumstance in which we would expect not to have found
	// a connector is if the auth service is the only service running in
	// this process.
	
->  is if the auth service or MDM service

Reply:
Done.